### ISSUE TITLE
Differentiable Sparse Linear Solve from Pytorch

### ISSUE THREAD

**Issue by ttaa9** [taichi-dev/taichi#301]
Hello! I have a pytorch pipeline, in the middle of which I'd like to be able to compute a sparse linear solve that I can backprop through. While Pytorch supports sparse tensors, this functionality [is currently lacking](https://github.com/pytorch/pytorch/issues/10043). 

Since I just noticed this library has some interoperability with pytorch, I was wondering if it would be possible to use difftaichi to compute this? 

Basically, something like

    Compute A, b as sparse pytorch tensors (batch of matrices & vectors)
    x = solve(A,b) # in taichi
    ... More pytorch ops ...
    loss.backward()

**Comment by belkakari** [comment]
Hi!
I have the same question, is it possible to incorporate taichi kernel as a part of the PyTorch graph with proper backpropagation?
